travis/publish.sh

```diff
@@ -72,7 +72,7 @@ check_release_tag() {
 }
 
 is_release_commit() {
-  project_version=$(./mvnw help:evaluate -N -Dexpression=project.version|grep -v '\[')
+  project_version=$(./mvnw help:evaluate -N -Dexpression=project.version|sed -n '/^[0-9]/p')
```

Comment:
Oh, one thing, though... this script allows `RC` releases...

```
"You must specify a tag of the format 'release-0.0.0' or 'release-0.0.0.RC0' to release this project."
```
https://github.com/opentracing/opentracing-java/pull/164/files#diff-f515214ad913b3a43604369f3a277d96R68